internal/blocktx/store/postgresql/migrations/000019_add_is_longest.up.sql

```diff
@@ -0,0 +1,13 @@
+-- field `is_longest` is an implementation detail that will help to
+-- make sure that there is only one longest chain at any given height
+-- and is also used as a helper when querying for longest chain
+ALTER TABLE blocktx.blocks
+ADD COLUMN is_longest BOOLEAN NOT NULL DEFAULT TRUE;
```

Comment:
Why is there the need for a field `is_longest` and `status`?
I think `status=10` means longest, so isn't `is_longest` redundant?